.github/workflows/ci.yml

```diff
@@ -30,13 +30,16 @@ jobs:
     uses: ./.github/workflows/lint.yml
   test:
     uses: ./.github/workflows/test.yml
+  prepare:
+    uses: ./.github/workflows/prepare-release.yml
```

Comment:
maybe this could be called `build` instead?

Reply:
updated